package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "another-todo-list-app",
+  "version": "0.0.0",
+  "scripts": {
+    "dev": "vite",
+    "build": "vite build",
+    "preview": "vite preview --port 4173",
+    "test:unit": "vitest --environment jsdom",
+    "test:e2e": "start-server-and-test preview http://localhost:4173/ 'cypress open --e2e'",
+    "test:e2e:ci": "start-server-and-test preview http://localhost:4173/ 'cypress run --e2e'",
+    "lint": "eslint . --ext .vue,.js,.jsx,.cjs,.mjs --fix --ignore-path .gitignore"
+  },
+  "dependencies": {
+    "vue": "^2.7.7",
+    "vue-router": "^3.5.4",
+    "vuex": "^3.1.1"
+  },
+  "devDependencies": {
+    "@rushstack/eslint-patch": "^1.1.0",
```

Comment:
@MpinaneMo You can import and quickly style your application using Tailwind classes. https://tailwindcss.com/